utils/docker/Dockerfile.el.8

```diff
@@ -8,8 +8,8 @@
 #
 
 # Pull base image
-ARG POINT_RELEASE=
-ARG BASE_DISTRO=rockylinux/rockylinux:8$POINT_RELEASE
+ARG RELEASE=8
```

Comment:
If we're doing this, why do we need separate `utils/docker/Dockerfile.el.8` and `utils/docker/Dockerfile.el.9` files?  Shouldn't we just have one `utils/docker/Dockerfile.el` file that includes conditional code for any differences between supported EL versions?  Otherwise, the `$POINT_RELEASE` makes sense as its relative to EL8.